deploy_docs.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+# Build the documentation from the SOURCE_BRANCH
+# and push it to TARGET_BRANCH.
+SOURCE_BRANCH="master"
+TARGET_BRANCH="gh-pages"
+
+# Pull requests and commits to other branches shouldn't try to deploy
+#if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_BRANCH" != "$SOURCE_BRANCH" ]; then
+#    echo "Skipping deploy on $TRAVIS_BRANCH. We only deploy docs automatically from development."
+#    exit 0
+#fi
+
+# Save some useful information
+REPO=`git config remote.origin.url`
+SSH_REPO=${REPO/https:\/\/github.com\//git@github.com:}
+SHA=`git rev-parse --verify HEAD`
+
+# Add rsa keys to the ssh agent to push to GitHub
```

Comment:
Is the commented out code supposed to be present? Like for users to uncomment (in which case do you want to add flags to the script instead), or is it old code that should be removed?

Reply:
The plan is to remove it, but I was waiting for Alice to come back and update her before we fully decide to abandon this method.

Reply:
Removed.